### PR TITLE
Add Edge metrics Vorto models.

### PR DESCRIPTION
**Pull request by dimitar-dimitrow**
Signed-off-by: Dimitar Dimitrov <dimitar.dimitrov3@bosch.io>

### CODE REVIEW

**Comment by dimitar-dimitrow** [comment]
@desislava-marinova, please wait for @e-grigorov to review the models before merging.

File: models/com.bosch.iot.suite.edge.metric-Filter-1.0.0.type/model.datatype [hunk 1]
```diff
@@ -0,0 +1,10 @@
+vortolang 1.0
+namespace com.bosch.iot.suite.edge.metric
+version 1.0.0
+displayname "Filter"
+description "Represents the required metrics per originator."
+
+entity Filter {
+  optional originator as string "The originator identifier."
```
Comment (e-grigorov) [inline models/com.bosch.iot.suite.edge.metric-Filter-1.0.0.type/model.datatype]: It'll be helpful to say what will happen when it's skipped as it's described for the measure identifiers.

File: models/com.bosch.iot.suite.edge.metric-Measurement-1.0.0.type/model.datatype [hunk 2]
```diff
@@ -0,0 +1,10 @@
+vortolang 1.0
+namespace com.bosch.iot.suite.edge.metric
+version 1.0.0
+displayname "Measurement"
+description "Represents a measured value per measure identifier."
+
+entity Measurement {
+  mandatory id as string "The measure identifier. For group of measures the syntax is <group>.<measure>, e.g. memory.total and memory.used."
```
Comment (e-grigorov) [inline models/com.bosch.iot.suite.edge.metric-Measurement-1.0.0.type/model.datatype]: `For group` should be: `For a group`

File: models/com.bosch.iot.suite.edge.metric-Metrics-1.0.0.fbmodel/model.functionblock [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+description "Provides functionality for reporting runtime metrics on a specified frequency."
+
+using com.bosch.iot.suite.edge.metric.OriginatorMeasurements;1.0.0
+using com.bosch.iot.suite.edge.metric.Filter;1.0.0
+
+functionblock Metrics {
+  events {
+    data {
+      mandatory multiple snapshot as OriginatorMeasurements "The snapshot with the measurements collected per each originator matching the request filter."
+      mandatory timestamp as long "The timestamp of metric data generation."
```
Comment (e-grigorov) [inline models/com.bosch.iot.suite.edge.metric-Metrics-1.0.0.fbmodel/model.functionblock]: `of metric data generation` should be: `of the metric data generation`

**Review by e-grigorov — CHANGES_REQUESTED** [review]

**Review by e-grigorov — APPROVED** [review]

**Pull request merged**